clang/test/dpct/helper_files_ref/include/ccl_utils.hpp

```diff
@@ -196,35 +201,52 @@ class communicator_ext : public dpct::ccl::detail::ccl_init_helper {
   oneapi::ccl::device _device_comm;
   oneapi::ccl::context _context_comm;
   oneapi::ccl::communicator _comm;
+  sycl::queue _queue;
```

Comment:
sycl::queue &

Reply:
no , if _queue defined by "sycl::queue &", it must be initialized in the constructor using an initializer list. 
But I want the queue to be saved when it is first used in member function.